packages/category-exporter/test/main.spec.js

```diff
@@ -0,0 +1,144 @@
+import streamtest from 'streamtest'
+import silentLogger from '../src/utils/silent-logger'
+import CategoryExporter from '../src/main'
+
+describe('CategoryExporter', () => {
+  const logger = {
+    ...silentLogger,
+  }
+
+  let categoryExport
+  beforeAll(() => {
+    categoryExport = new CategoryExporter({
+      apiConfig: {
+        projectKey: 'my-test-project',
+      },
+      logger,
+    })
+  })
+
+  describe('::constructor', () => {
+    test('should be a function', () => {
+      expect(typeof CategoryExporter).toBe('function')
+    })
+
+    test('should set default properties', () => {
+      expect(categoryExport.apiConfig).toEqual({
+        projectKey: 'my-test-project',
+      })
+      expect(categoryExport.logger).toEqual(logger)
+    })
+
+    test('should throw error if no `apiConfig` in `options` parameter', () => {
+      expect(() => new CategoryExporter({ froo: 'bar' })).toThrowError(
+        /The constructor must be passed an `apiConfig` object/
+      )
+    })
+  })
+
+  describe('::run ', () => {
+    describe('with status code 200', () => {
+      let payload
+      beforeAll(() => {
+        payload = {
+          statusCode: 200,
+          body: {
+            results: [{ id: 'id1', name: 'foo' }, { id: 'id2', name: 'bar' }],
+          },
+        }
+        categoryExport.client.execute = jest
+          .fn()
+          .mockReturnValue(Promise.resolve(payload))
+      })
+      test('should be a function', () => {
+        expect(typeof categoryExport.run).toBe('function')
+      })
+
+      test('should write to outputStream', done => {
+        const outputStream = streamtest.v2.toText((error, data) => {
+          expect(error).toBeFalsy()
+          expect(data).toEqual(JSON.stringify(payload.body.results))
+          done()
+        })
+        categoryExport.run(outputStream)
+      })
+    })
+
+    describe('with status code 404', () => {
+      let results
+      beforeAll(() => {
+        results = {
+          statusCode: 404,
+          body: {
+            results: [],
+          },
+        }
+        categoryExport.client.execute = jest
+          .fn()
+          .mockReturnValue(Promise.resolve(results))
+      })
+
+      test('should write to outputStream', done => {
```

Comment:
I dont think `should write to outputStream` is the best description here. Maybe something like `should throw error`?